pkg/operator/secretannotator/equinixmetal/reconciler.go

```diff
@@ -0,0 +1,183 @@
+/*
```

Comment:
Mostly copy/paste from `pkg/operator/secretannotator/vsphere/reconciler.go`

Reply:
Same thing here. We don't need in-cluster processing of the Secret that idealy should not even exist.